prepare-cluster/action.yml

```diff
@@ -215,7 +215,7 @@ runs:
     run: node ${{ github.action_path }}/kots-install.js
     env:
       INPUT_KUBECONFIG: ${{ steps.create-cluster.outputs.cluster-kubeconfig }}
-      INPUT_APP-SLUG: ${{ inputs.app-slug }}/${{ steps.create-release.outputs.channel-slug }}
+      INPUT_APP-SLUG: ${{ inputs.app-slug }}/test-channel
```

Comment:
is this intentional?

Reply:
Yes. A customer with license type `test` has no channel assigned. The promoted release is actually on the hidden `test-channel`.